backend/app/models/llm/request.py

```
@@ -349,23 +433,21 @@ class ProxyCompletionConfig(SQLModel):
-
 
 # Discriminated union for completion configs based on provider field
 CompletionConfig = Annotated[
-    Union[NativeCompletionConfig, KaapiCompletionConfig, ProxyCompletionConfig],
+    Union[
+        NativeCompletionConfig,
+        KaapiTextCompletionConfig,
+        KaapiSTTCompletionConfig,
+        KaapiTTSCompletionConfig,
+        ProxyCompletionConfig,
+    ],
```

Ruff (0.16.2)

[warning] Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)